src/components/TeamMember.js

```diff
@@ -0,0 +1,33 @@
+import {
+  Box,
+  Card,
+  CardContent,
+  Typography,
+} from "@material-ui/core/";
```

Comment:
If I remember correctly you could import like `import Box from "@material-ui/core/Box";` - it support tree-shaking, more here:
https://material-ui.com/guides/minimizing-bundle-size/

change is not needed, it's just a tip ;)

Reply:
Project's webpack version supports tree-shaking but noted to be aware of it in other projects :)